src/Symfony/Component/Form/Tests/AbstractBootstrap3LayoutTest.php

```diff
@@ -968,8 +960,7 @@ public function testMultipleChoiceExpandedAttributes()
         $form = $this->factory->createNamed('name', 'Symfony\Component\Form\Extension\Core\Type\ChoiceType', array('&a', '&c'), array(
             'choices' => array('Choice&A' => '&a', 'Choice&B' => '&b', 'Choice&C' => '&c'),
             'choice_attr' => array('Choice&B' => array('class' => 'foo&bar')),
-            'multiple' => true,
-            'expanded' => true,
+            'widget' => 'checkbox',
             'required' => true,
```

Comment:
Some tests for `widget` as `text` or `hidden` rendering with bootstrap and different choice type options being set should be added IMO.

You could use something like `testSingleSimpleChoice*`, `testMultipleSimpleChoices*`, `testSingleSimpleChoiceHidden*` and `testMultipleSimpleChoicesHidden*` as prefix.